src/main/java/cz/geokuk/plugins/mrizky/JMrizkaDdMmMmm.java

```diff
@@ -22,12 +22,12 @@ public JSingleSlide0 createRenderableSlide() {
 
 	@Override
 	public String getTextX(final double x) {
-		return Wgs.toGeoFormat(x);
+		return (x < 0 ? "-" : "") + Wgs.toGeoFormat(Math.abs(x));
```

Comment:
Šlo mi o minimální zásah do toho, jak Geokuk funguje teď. Nyní je v mřížce souřadnice jenom jako číslo bez N nebo E. Jistě by to šlo udělat.